kaldb/src/main/java/com/slack/kaldb/chunk/ChunkInfo.java

```diff
@@ -66,77 +66,84 @@ public void setChunkSize(long chunkSize) {
   // Size of the chunk
   private long chunkSize;
 
-  public ChunkInfo(String chunkId, long chunkCreationTimeEpochSecs) {
+  public ChunkInfo(String chunkId, long chunkCreationTimeEpochMs) {
     ensureTrue(chunkId != null && !chunkId.isEmpty(), "Invalid chunk dataset name " + chunkId);
     ensureTrue(
-        chunkCreationTimeEpochSecs >= 0,
-        "Chunk creation time should be non negative: " + chunkCreationTimeEpochSecs);
+        chunkCreationTimeEpochMs >= 0,
+        "Chunk creation time should be non negative: " + chunkCreationTimeEpochMs);
 
     this.chunkId = chunkId;
-    this.chunkCreationTimeEpochSecs = chunkCreationTimeEpochSecs;
-    dataStartTimeEpochSecs = 0;
-    dataEndTimeEpochSecs = 0;
-    chunkLastUpdatedTimeSecsEpochSecs = chunkCreationTimeEpochSecs;
+    this.chunkCreationTimeEpochMs = chunkCreationTimeEpochMs;
+    dataStartTimeEpochMs = 0;
+    dataEndTimeEpochMs = 0;
+    chunkLastUpdatedTimeEpochMs = chunkCreationTimeEpochMs;
     // TODO: Should we set the snapshot time to creation time also?
-    chunkSnapshotTimeEpochSecs = 0;
+    chunkSnapshotTimeEpochMs = 0;
   }
 
-  public long getChunkSnapshotTimeEpochSecs() {
-    return chunkSnapshotTimeEpochSecs;
+  public long getChunkSnapshotTimeEpochMs() {
+    return chunkSnapshotTimeEpochMs;
   }
 
-  public void setChunkSnapshotTimeEpochSecs(long chunkSnapshotTimeEpochSecs) {
-    this.chunkSnapshotTimeEpochSecs = chunkSnapshotTimeEpochSecs;
+  public void setChunkSnapshotTimeEpochMs(long chunkSnapshotTimeEpochMs) {
+    this.chunkSnapshotTimeEpochMs = chunkSnapshotTimeEpochMs;
   }
 
-  public long getDataStartTimeEpochSecs() {
-    return dataStartTimeEpochSecs;
+  public long getDataStartTimeEpochMs() {
+    return dataStartTimeEpochMs;
   }
 
-  public long getDataEndTimeEpochSecs() {
-    return dataEndTimeEpochSecs;
+  public long getDataEndTimeEpochMs() {
+    return dataEndTimeEpochMs;
   }
 
-  public long getChunkCreationTimeEpochSecs() {
-    return chunkCreationTimeEpochSecs;
+  public long getChunkCreationTimeEpochMs() {
+    return chunkCreationTimeEpochMs;
   }
 
-  public long getChunkLastUpdatedTimeSecsEpochSecs() {
-    return chunkLastUpdatedTimeSecsEpochSecs;
+  public long getChunkLastUpdatedTimeEpochMs() {
+    return chunkLastUpdatedTimeEpochMs;
   }
 
-  public void setChunkLastUpdatedTimeSecsEpochSecs(long chunkLastUpdatedTimeSecsEpochSecs) {
-    this.chunkLastUpdatedTimeSecsEpochSecs = chunkLastUpdatedTimeSecsEpochSecs;
+  public void setChunkLastUpdatedTimeEpochMs(long chunkLastUpdatedTimeEpochMs) {
+    this.chunkLastUpdatedTimeEpochMs = chunkLastUpdatedTimeEpochMs;
+  }
+
+  public void setDataStartTimeEpochMs(long dataStartTimeEpochMs) {
```

Comment:
Can we drop these set methods since they are unused? Also, I think removing the fields also makes these values set only which may be an advantage.